backstop/tests/modal.html

```diff
@@ -62,37 +86,141 @@ <h1>Modal</h1>
+            eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad
+            minim veniam, quis nostrud exercitation ullamco laboris nisi ut
+            aliquip ex ea commodo consequat. Duis aute irure dolor in
+            reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla
+            pariatur. Excepteur sint occaecat cupidatat non proident, sunt in
+            culpa qui officia deserunt mollit anim id est laborum.
+          </p>
+        </div>
         <div class="iui-button-bar">
           <button
             class="iui-button iui-high-visibility"
-            onclick="openModal()"
+            onclick="toggleModal('default-modal')"
           >
             <span class="iui-button-label">Proceed</span>
           </button>
           <button
             class="iui-button iui-default"
-            onclick="openModal()"
+            onclick="toggleModal('default-modal')"
           >
             <span class="iui-button-label">Cancel</span>
           </button>
         </div>
       </div>
     </div>
+
+    <div
+      class="iui-modal iui-modal-full-page iui-modal-full-page-slide-animation"
+      id="full-page-modal-slide"
+    >
+      <div class="iui-modal-dialog">
+        <div class="iui-title-bar">
+          <div class="iui-title">Title</div>
+          <button
+            aria-label="Close"
+            type="button"
+            class="iui-button iui-borderless iui-small"
+            onclick="toggleModal('full-page-modal-slide')"
+          >
+            <svg-close
+              class="iui-button-icon"
+              aria-hidden="true"
+            ></svg-close>
+          </button>
+        </div>
+        <div class="iui-modal-content">
+          <p>
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+            eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad
+            minim veniam, quis nostrud exercitation ullamco laboris nisi ut
+            aliquip ex ea commodo consequat. Duis aute irure dolor in
+            reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla
+            pariatur. Excepteur sint occaecat cupidatat non proident, sunt in
+            culpa qui officia deserunt mollit anim id est laborum.
+          </p>
+        </div>
+        <div class="iui-button-bar">
+          <button
+            class="iui-button iui-high-visibility"
+            onclick="toggleModal('full-page-modal-slide')"
+          >
+            <span class="iui-button-label">Proceed</span>
+          </button>
+          <button
+            class="iui-button iui-default"
+            onclick="toggleModal('full-page-modal-slide')"
+          >
+            <span class="iui-button-label">Cancel</span>
+          </button>
+        </div>
+      </div>
+    </div>
+
+    <div
+      class="iui-modal iui-modal-full-page iui-modal-full-page-scale-animation"
```

Comment:
Let's remove the scaling transition & only use the slide up from the bottom transition.  That way `iui-modal-full-page` can include the transition within it & no need for an extra modifier class like `.iui-modal-full-page-scale-animation`.

Reply:
I did it, but are you sure? You really liked the scale. I can include it in react also.

Reply:
Yeah, just for simplicity let's only support 1 option for initial release.  We can think about adding other options in the future.